packages/uploadthing/src/next.ts

```diff
@@ -21,52 +17,14 @@ export const createUploadthing = <TErrorShape extends Json>(
 export const createNextRouteHandler = <TRouter extends FileRouter>(
```

Comment:
this doesn't actually need to exist, we could alias it to `createServerHandler` and potentially deprecate it in favor of

```ts
import { createServerHandler } from "uploadthing/server";
import { createUploadThing } from "uploadthing/next";
```